test/specs/reference/tests/11.delete.js

```diff
@@ -4,14 +4,14 @@ exports.execute = function (options) {
         var catalogId = process.env.DEFAULT_CATALOG,
             schemaName = "delete_schema",
             tableName = "delete_table",
+            tuples = [{data: {key_col: 1}}],
```

Comment:
It's better if you actually read and pass `page.tuples` to the function (instead of hacking the tuple object). 

We also need two more test cases. One for the case of passing multiple tuples and another for calling delete without any parameters.
We could also test this with a shortestkey of length >1, but I don't think that's needed (soon all deployments will enforce RID).